tests/cmds/object_files/coinrecords/coinrecord.json

```diff
@@ -1,7 +1,11 @@
-{"coin": {"amount": 0,
-           "parent_coin_info": "0x0000000000000000000000000000000000000000000000000000000000000000",
-           "puzzle_hash": "0000000000000000000000000000000000000000000000000000000000000000"},
-  "coinbase": true,
-  "confirmed_block_index": 1,
-  "spent_block_index": 1,
-  "timestamp": 909469800}
+{
+   "coin":{
+      "parent_coin_info":"0x0000000000000000000000000000000000000000000000000000000000000000",
+      "puzzle_hash":"0x0000000000000000000000000000000000000000000000000000000000000000",
+      "amount":0
+   },
+   "confirmed_block_index":1,
+   "spent_block_index":1,
+   "coinbase":true,
+   "timestamp":909469800
+}
```

Comment:
Can you elaborate why you made these changes?  The point behind them being different was to test that the inspect command was working correctly with or without `0x`

Reply:
With 1.5.1 tests fail without these changes

Reply:
That would be indicative of a error then, and we shouldn't just abandon the flexibility right?

Reply:
Ah, let me try to re add the functionality

Reply:
i did not realize it was intended.

Reply:
uh @Quexington https://github.com/Chia-Network/chia_rs/blob/f5fdb37b8c62889695b76ccc9a1fb76caabf1c48/wheel/src/from_json_dict.rs#L18

Reply:
Okay I'm curious why that is in there but I suppose I'll defer to the upstream